Arrowgene.Ddon.Shared/Network/PacketId.cs

```diff
@@ -923,7 +923,9 @@ public override int GetHashCode()
         public static readonly PacketId C2S_WARP_GET_START_POINT_LIST_REQ = new PacketId(14, 14, 1, "C2S_WARP_GET_START_POINT_LIST_REQ", ServerType.Game, PacketSource.Client);
         public static readonly PacketId S2C_WARP_GET_START_POINT_LIST_RES = new PacketId(14, 14, 2, "S2C_WARP_GET_START_POINT_LIST_RES", ServerType.Game, PacketSource.Server); // ゲーム開始ポイントの取得に
 
-// Group: 15
+        // Group: 15
+        public static readonly PacketId C2S_15_0_16_NTC = new PacketId(15, 0, 16, "C2S_15_0_16_NTC", ServerType.Game, PacketSource.Client); // Appears to be cutscene related (before)
+        public static readonly PacketId C2S_15_1_16_NTC = new PacketId(15, 1, 16, "C2S_15_1_16_NTC", ServerType.Game, PacketSource.Client); // Appears to be cutscene related (after)
```

Comment:
Chances are these are C2S_EVENT_START_NTC and C2S_EVENT_END_NTC

Reply:
Do you want me to rename them to that?

Reply:
Yes, please, i already had them noted down but didn't add the names since they weren't in the code, but now that we may begin to have use for them it'd be good to have them labelled